.github/workflows/sonarqube_action.yaml

```
@@ -0,0 +1,52 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v5
+        with:
+            fetch-depth: 0 # Required for SonarQube analysi
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)